Doc/library/threading.rst

```diff
@@ -16,9 +16,9 @@ level :mod:`_thread` module.  See also the :mod:`queue` module.
 
 .. note::
 
-   While they are not listed below, the ``camelCase`` names used for some
+   The ``camelCase`` names used for some
    methods and functions in this module in the Python 2.x series are still
-   supported by this module.
+   supported by this module for compatibility with Python 2.5 and lower.
```

Comment:
I would prefer to say "deprecated" somewhere, rather than "still supported".

Reply:
Reworded